tests/common.go

```diff
@@ -533,6 +516,11 @@ func PerformSystemCheck() {
 				expect(file).To(beEmpty())
 			}
 		})
+		ginkgo.JustAfterEach(func() {
```

Comment:
Take the changes for PerformSystemCheck() out of this PR and put it in a separate PR.